devel/xcode-build-server/Portfile

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           python 1.0
+
+github.setup        SolaWing xcode-build-server 1.0.0 v
+version             1.0.0
+revision            0
+license             MIT
+categories          devel python
+maintainers         {woolsweater.net:macports @woolsweater} \
+                    openmaintainer
+supported_archs     noarch
+
+description         Build Server Protocol implementation for Xcode projects
+
+long_description    ${name} integrates Xcode with Apple's sourcekit-lsp. \
+                    sourcekit-lsp doesn't itself support Xcode projects, but it does \
+                    provide a Build Server Protocol client to work with other \
+                    build systems. xcode-build-server implements a build \
+                    server to provide sourcekit-lsp with the build \
+                    information it needs for an Xcode project.
+
+checksums           rmd160  fd21bf312b47c15fc28d03b80de1ec39d76140a2 \
+                    sha256  08ea2b9a892670bb7c2f8ff90a97ae0be519744f42ba713bb1a56e247c77e885 \
+                    size    18673
+
+build {}
+
+post-patch {
+    reinplace "s|#!/usr/bin/env python3|#!/usr/bin/env python${python.branch}|" \
```

Comment:
please do the reinplace to change it to "${python.bin}"

Reply:
`${python.bin}` results in `#!/usr/bin/env python/opt/local/Library/Frameworks/Python.framework/Versions/3.12/bin/python3.12`, which doesn't execute.

Reply:
what I mean is that you should replace "/usr/bin/env python3" with "{python.bin}", that will work

Reply:
Okay, got it.

Reply:
so, this should become:
```
post-patch {
    reinplace "s|/usr/bin/env python3|${python.bin}|" ${worksrcpath}/${name}
}
```